src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item {
+    list-style: none;
+    text-transform: uppercase;
```

Comment:
The 'list-style: none;' property is not necessary here since it's already defined in the '&__list' element.